packages/geo-tools/package.json

```diff
@@ -0,0 +1,115 @@
+{
+  "name": "@bentley/geo-tools-react",
+  "version": "0.0.1",
+  "description": "React Geospatial Tools",
+  "keywords": [
+    "Bentley",
+    "iModel.js",
+    "Geo",
+    "Bing",
+    "Map",
+    "Search",
+    "Address"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/geo-tools"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/geo-tools.js",
+  "typings": "lib/geo-tools",
+  "scripts": {
+    "build": " tsc 1>&2 && npm run copy:assets && npm run pseudolocalize",
+    "build-prod": " tsc -p tsconfig.json 1>&2 && npm run copy:assets",
+    "clean": "rimraf lib",
+    "copy:assets": "cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
+    "cover": "nyc npm test",
+    "lint": "eslint ./src/**/*.{ts,tsx} 1>&2",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "test": "mocha --opts ./mocha.opts \"./lib/test/**/*.test.js\""
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "i18next": "^10.2.2",
+    "react-autosuggest": "^10.0.2"
+  },
+  "devDependencies": {
+    "@bentley/build-tools": "2.10.0-dev.5",
+    "@bentley/eslint-plugin": "2.10.0-dev.5",
+    "@bentley/itwin-client": "2.10.0-dev.5",
+    "@bentley/telemetry-client": "2.10.0-dev.5",
```

Comment:
missing quite a few dev deps, any of the core imjs peer deps should be listed here as dev deps too

Reply:
i dont think you need telemetry-lcient, so can remove that. need to add geom-core, bentleyjs-core, ui-abstract, ui-frmework, ui-core, imjs-common, imjs-frontend, imjs-i18n